.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/digitalbitbox/yajl.git
 [submodule "secp256k1"]
 	path = src/secp256k1
-	url = https://github.com/digitalbitbox/secp256k1.git
+	url = https://github.com/digitalbitbox/secp256k1_patch.git
```

Comment:
I cleaned up the commit history in the secp repository. This entailed a force push, so I archived the existing repository, in order for older releases of the firmware to still be built, and created a new 'patch' repository to link to.